LATEST

```diff
@@ -1,3 +1,4 @@
+c4d82f724c78d1f97ab462b7fc2aad3c786e972c 2.0.0-snapshot.20211207.8608.0.c4d82f72
 85e2fa551c6a607d2f59b44bed8b667c1e577322 2.0.0-snapshot.20211207.8589.3.85e2fa55 SPLIT_RELEASE
 1d0ff3cc71583f738489e7732fa758a39f9cbfbd 2.0.0-snapshot.20211201.8538.0.1d0ff3cc
```

Comment:
```suggestion
85e2fa551c6a607d2f59b44bed8b667c1e577322 2.0.0-snapshot.20211207.8589.3.85e2fa55 SPLIT_RELEASE
c4d82f724c78d1f97ab462b7fc2aad3c786e972c 2.0.0-snapshot.20211207.8608.0.c4d82f72
```
We usually keep one line per stable release (and one for the new split release stuff but that’s hopefully temporary) rathre than adding it. Both work, it’s just convention.

Reply:
ah thanks!